### DISCUSSION TITLE
Disable taskbar thumbnails ...

### DISCUSSION THREAD

**Discussion by michaelmuselik** [valinet/ExplorerPatcher#1476] [Ideas]
Any function solution? I tested all known solution, but no known solution works permanently. 
Thnx

**Comment by pyrates999** [comment]
This might help: https://www.thewindowsclub.com/tenable-or-disable-taskbar-thumbnail-preview-in-windows-10

**Reply by michaelmuselik** [reply]
@pyrates999 thanks for sharing tips. I have tried all of them, but none work long for a long term, permanently.

**Comment by pyrates999** [comment]
Check out 7+ Taskbar Tweaker: https://ramensoftware.com/7-taskbar-tweaker

It has an option to disable taskbar thumbnails.

Look under "Hovering" for it.

It does not run under Windows 11 though.

**Reply by michaelmuselik** [reply]
Hi @pyrates999 thanx for tip 👍. I know this app, and it is good, but I'm looking for a solution **without the need for another application**. Ideally integrate this function in [EP](https://github.com/valinet/ExplorerPatcher)?? Is it possible? @valinet

**Reply by pyrates999** [reply]
I don't see why you can't install another app if it's compatible with your version of windows to do what you want.  Since you don't want to install it, all you can do is wait and see if it actually gets added.
